src/tests/comments.test.ts

```diff
@@ -86,9 +89,9 @@ describe("POST /comments", () => {
   });
 });
 
-let comments = [
-  { content: "First comment", sender: "amitinbar" },
-  { content: "Second comment", sender: "amitinbar" },
+let comments: any = [
```

Comment:
```suggestion
let comments: Partial<ICommentDocument>[] = [
```

This is a possible type